rudder-core/src/main/scala/com/normation/rudder/services/policies/write/AgentSpecificLogic.scala

```diff
@@ -161,7 +155,7 @@ object CFEngineAgentSpecificGeneration extends AgentSpecificGeneration {
   }
 
   override def write(cfg: AgentNodeWritableConfiguration): Box[List[AgentSpecificFile]] = {
-    writeExpectedReportsCsv(cfg.paths, cfg.expectedReportsCsv, GENEREATED_CSV_FILENAME)
+    Full(Nil)
```

Comment:
Could you had a comment referencing the ticket, something like: 
# We believe that expectedReports.csv is not used anymore on agent but still for expecter report generation.
# This is the first step to check that our hypothesis is ok before removing totally: https://www.rudder-project.org/redmine/issues/12225

Reply:
SHould i write iit this big ? :)